openapi.yaml

```diff
@@ -0,0 +1,468 @@
+      responses:
+        '200':
+          description: The table retrieved successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/table'
+    put:
+      summary: Edit table
+      tags:
+        - table
+      parameters: 
+        - name: id
+          in: path
+          schema:
+            $ref: '#/components/schemas/mongoid'
+          required: true
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/table'
+      responses:
+        '200':
+          description: The table changed successfully
+    delete:
+      summary: Delete table
+      tags:
+        - table
+      parameters: 
+        - name: id
+          in: path
+          required: true
+          schema:
+            $ref: '#/components/schemas/mongoid'
+      responses:
+        '200':
+          description: The table deleted successfully
+  /image/all:
+    get:
+      summary: Get all images
+      tags:
+        - image
+      responses:
+        '200':
+          description: The images retrieved successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/image'
+  /image:
+    post:
+      summary: Create a new image
+      tags:
+        - image
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/image'
+      responses:
+        '200':
+          description: The image created succesfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/image'
+  /image/{name}:
+    get:
+      summary: Get image
+      tags:
+        - image
+      parameters:
+        - name: name
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: The image retrieved successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/image'
+    delete:
+      summary: Delete image
+      tags:
+        - image
+      parameters: 
+        - name: name
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: The image deleted successfully
+  /settings/{settingName}:
+    get:
+      summary: Get setting value
+      tags:
+        - settings
+      parameters:
+        - name: settingName
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: The setting retrieved successfully
+          content:
+            application/json:
+              schema: {}
+              example: anything
+    delete:
```

Comment:
I dont think there is a delete function for setting

Reply:
Isn't it used for the Trashcan button?
![image](https://user-images.githubusercontent.com/6757517/65405407-a3afcc00-dde4-11e9-9c07-e8a606935653.png)

Reply:
@alanggreen but not on settings